### PR TITLE
Moe ref

### PR DESCRIPTION
**Pull request by sljlp**
### PR types

Others
### PR changes

Docs
### Describe

add some moe references

### CODE REVIEW

**Review by gongweibao — APPROVED** [review]
LGTM

File: paddle/fluid/operators/assign_pos_op.cu [hunk 1]
```diff
@@ -10,7 +10,11 @@ Unless required by applicable law or agreed to in writing, software
 distributed under the License is distributed on an "AS IS" BASIS,
 WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
-limitations under the License. */
+limitations under the License.
+
+Referenced codes:
```
Comment (gongweibao) [inline paddle/fluid/operators/assign_pos_op.cu]: These codes are modified from xxx

**Review by gongweibao — APPROVED** [review]

**Review by gongweibao — CHANGES_REQUESTED** [review]
These codes are modified from xxx and add copyright of them

**Review by ForFishes — DISMISSED** [review]
LGTM

**Review by ForFishes — DISMISSED** [review]
LGTM

File: python/paddle/incubate/distributed/models/moe/gate/naive_gate.py [hunk 2]
```diff
@@ -11,6 +11,13 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+#
+# The file has been adapted from the file:
+#     https://github.com/laekov/fastmoe/blob/master/fmoe/gates/gshard_gate.py
```
Comment (gongweibao) [inline python/paddle/incubate/distributed/models/moe/gate/naive_gate.py:16]: naive_gate?

**Review by gongweibao — APPROVED** [review]
LGTM

**Pull request merged**